src/main.c

```diff
@@ -28,6 +28,7 @@ static const struct cmdarg arguments[] = {
     { 'g', "gallery",    NULL,    "start in gallery mode" },
     { 'r', "recursive",  NULL,    "read directories recursively" },
     { 'o', "order",      "ORDER", "set sort order for image list: none/[alpha]/reverse/random" },
+    { 'R', "reverse",    NULL,    "reverse sort order" },
```

Comment:
This change requires modification of man and bash/zsh completion scripts.
It's not used very often and can be set via `-c`, so it can be omitted.